tests/testthat/_snaps/fit_model.md

```diff
@@ -0,0 +1,8 @@
+# Right truncation longer than data throws error
+
+    Removing right-truncation PMF elements after 1
```

Comment:
After 1? But PMF is 3 and data is 2. Shouldn't it be after 2?

Reply:
I did one less for safety because I wasn't sure if it would be finicky with exactly equal length.

Reply:
I guess I'm fine with "remove one more for safety" but do we actually know that equal length is a problem?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/55/commits/382dd0bc9c6f9658d618a10a572a35fc8c92933b

Reply:
"No margin for error" is a very metal name